src/core/src/op/constant.cpp

```diff
@@ -43,12 +43,23 @@ std::string to_cpp_string(T value) {
     }
 }
 
-std::vector<double> from_string_vector(const std::vector<std::string>& str_values) {
-    std::vector<double> values;
+template <class T, typename std::enable_if<std::is_integral<T>::value>::type* = nullptr>
+T str_to_value(const std::string& s, size_t* pos) {
+    return static_cast<T>(std::is_signed<T>::value ? std::stol(s, pos) : std::stoul(s, pos));
+}
```

Comment:
What about having it separate?
```suggestion
template <class T,
          typename std::enable_if<std::is_integral<T>::value>::type* = nullptr,
          typename std::enable_if<std::is_signed<T>::value>::type* = nullptr>
T str_to_value(const std::string& s, size_t* pos) {
    return static_cast<T>(std::stol(s, pos));
}

template <class T,
          typename std::enable_if<std::is_integral<T>::value>::type* = nullptr,
          typename std::enable_if<std::is_unsigned<T>::value>::type* = nullptr>
T str_to_value(const std::string& s, size_t* pos) {
    return static_cast<T>(std::stoul(s, pos));
}
```

Reply:
As for internal function there is not much benefit re-work it, Will keep current solution (the inside if is well optimized as value is resolved during compile time).